warehouse/oidc/forms/google.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+import wtforms
+
+from warehouse import forms
+from warehouse.i18n import localize as _
+from warehouse.oidc.forms._core import PendingPublisherMixin
+
+_VALID_GITHUB_REPO = re.compile(r"^[a-zA-Z0-9-_.]+$")
+_VALID_GITHUB_OWNER = re.compile(r"^[a-zA-Z0-9][a-zA-Z0-9-]*$")
+
+
+class GooglePublisherBase(forms.Form):
+    __params__ = ["email", "sub"]
+
+    email = wtforms.fields.EmailField(
+        validators=[
+            wtforms.validators.InputRequired(),
+            wtforms.validators.Regexp(
+                r".+@.+\..+", message=_("The email address isn't valid. Try again.")
+            ),
```

Comment:
I realize this is probably copy&pasta from `NewEmailMixin`, but could this be replaced with a built-in validator? https://wtforms.readthedocs.io/en/3.0.x/validators/#wtforms.validators.Email

We have the required packages already, and use the validator elsewhere in the codebase.

```suggestion
            wtforms.validators.Email(),
```

Reply:
Aha, I think this is a holdover from before we started using that (this PR is quite old). Nice catch.